requirements.txt

```diff
@@ -1,3 +1,4 @@
 fastapi >=0.100
 uvicorn[standard] >=0.29.0
 pyzmq >=22.0.0
+orjson==3.10.15
```

Comment:
let's keep this dependency optional

Reply:
Sure, right now I'm adding it to the test and perf requirements in the `_requirements` directory. Is there a better place for it?